src/main/java/seedu/address/commons/events/ui/SwitchToToDoTabEvent.java

```diff
@@ -5,7 +5,7 @@
 /**
  * An event requesting to switch to the To Do List tab
  */
-public class SwitchTabEvent extends BaseEvent {
```

Comment:
Actually I think this should remain "SwitchTabEvent", or else maybe change to "ToggleTabEvent", because it is supposed to toggle both ways (from search to todo, or from todo to search).

Reply:
Oh ya you are right! I mixed it up with the other 2 events. Ill change it back